content/docs/start/experiments.md

```diff
@@ -18,17 +18,90 @@ once they're no longer needed.
 
 ## Running experiments
 
-Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
-[compare the changes](/doc/start/metrics-parameters-plots). Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+In the previous page, we learned how to tune
+[ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section
+we'll use a
+[Get Started with Experiments project](https://github.com/iterative/get-started-experiments)
+to illustrate experimentation features in DVC 2.0.
```

Comment:
> Cont https://github.com/iterative/dvc.org/pull/2491#discussion_r637487008

```suggestion
[compare the changes](/doc/start/metrics-parameters-plots). Let's now
illustrate the experimentation features in DVC 2.0 with a different project
[example](https://github.com/iterative/get-started-experiments).
```
May not even need to link here, it can be just inside the `<details>` section below. Let's focus on concepts and explaining 🙂

Reply:
I'm using the links in both places in #2574.